src/lib/fees/utils.ts

```diff
@@ -0,0 +1,51 @@
+import dayjs from 'dayjs'
+import duration from 'dayjs/plugin/duration'
+import { getEthersProvider } from 'lib/ethersProviderSingleton'
+
+dayjs.extend(duration)
+const ETHEREUM_AVERAGE_BLOCK_TIME_SECONDS = 12.08
+
+export const findClosestFoxDiscountDelayBlockNumber = async (
+  delayHours: number,
+): Promise<number> => {
+  const currentBlock = await getEthersProvider().getBlockNumber()
+  if (delayHours === 0) return currentBlock
+  const dayjsDelay = dayjs.duration(delayHours, 'hours')
+  const targetTimestamp = dayjs().subtract(dayjsDelay).unix()
```

Comment:
can't you just use `dayjs().subtract(delayHours, 'hours').unix()` without the duration boilerplate?

Reply:
absolutely - 2bd07a99c5